src/Sentry/Laravel/EventHandler.php

```diff
@@ -291,6 +291,8 @@ protected function httpClientResponseReceivedHandler(HttpClientEvents\ResponseRe
                 'status_code' => $event->response->status(),
                 'http.query' => $fullUri->getQuery(),
                 'http.fragment' => $fullUri->getFragment(),
+                'request_body_size' => strlen($event->request->body()),
+                'response_body_size' => strlen($event->response->body()),
```

Comment:
request/response body size is in bytes instead of string length.

Reply:
See: https://www.php.net/manual/en/function.strlen.php#refsect1-function.strlen-notes

Reply:
Yep, my bad, I just assumed because its called `strlen`, I know nothing about PHP :D

Reply:
Easy mistake to make, welcome in PHP land 😉